pyEpiabm/pyEpiabm/sweep/host_progression_sweep.py

```diff
@@ -52,22 +66,50 @@ def _set_latent_time(self, person, time):
 
         person.time_of_status_change = time + latent_time
 
+    def _set_infectiousness(self, person):
+        """Assigns the infectiousness of a person for when they go from
+        the exposed infection state to the next state, either InfectAsympt,
+        InfectMild or InfectGP.
+        *Needs to be called right after an exposed person has been given its
+        new infection status in the sweep*
```

Comment:
Presumably this is called internally if its a private method, so this seems slightly redundant - maybe detail where it is called from?